src/Controller/ResultController.php

```diff
@@ -14,11 +15,21 @@
 class ResultController extends AbstractController
 {
```

Comment:
If you add property docblocks let us be consistent and add them to all properties